docs/usage/switching-bot-identity.md

```diff
@@ -0,0 +1,52 @@
+# Switching bot identity
+
+Renovate uses its bot identity to know:
+
+- if a PR is authored by Renovate bot
+- if you, or some other bot, pushed commits into the PR branch
+- Editor note: list other things Renovate uses the bot id for here
+
+## Reasons to switch bot identity
+
+Common reasons to switch bot identity are:
+
+- Migrating between Mend-hosted and self-hosted
+- Renaming the bot (self-hosted)
+- Editor note: Maybe there are other reasons too, please comment
+
+### Migrating from Mend-hosted to self-hosted
+
+Most users start with the Mend-hosted bot, and are happy with this.
+After a while, some users want to switch to self-hosting, for more control.
+
+When you migrate you must tell your new self-hosted Renovate bot to take over from the old Mend-hosted bot.
+
+### Renaming the bot (self-hosted)
+
+Maybe you started self-hosting Renovate, and called your bot `@old-bot-name`.
+But the name no longer fits, so you want to use `@new-bot-name`.
+
+Follow these steps:
+
+1. Start of ordered list with steps
+1. Second step
+1. Third step, and so on
+
+### Other situations
+
+Editor note: Please comment about other cases where you need to switch bot identity.
```

Comment:
```suggestion
```

If there are no more situations where a user may want/need to switch the Renovate bot ID, we can drop the whole section with this suggestion. 😉